src/number_theory/cyclotomic/primitive_roots.lean

```diff
@@ -193,6 +202,154 @@ begin
       ring_hom.eq_int_cast, int.cast_id]
 end
 
-end norm
+/-- If `is_prime_pow (n : ℕ)`, `n ≠ 2` and `irreducible (cyclotomic n K)` (in particular for
+`K = ℚ`), then the norm of `ζ - 1` is `(n : ℕ).min_fac`. -/
+lemma is_prime_pow_sub_one_norm (hn : is_prime_pow (n : ℕ)) [is_cyclotomic_extension {n} K L]
```

Comment:
I'm gonna suggest a different naming for this because I really don't like these:

```suggestion
lemma sub_one_norm.is_prime_pow (hn : is_prime_pow (n : ℕ)) [is_cyclotomic_extension {n} K L]
```